benchmarks/python/test_embedding.py

```diff
@@ -86,3 +94,66 @@ def fwd_call(inp):
         [outputs[0], test_case.grads(), *list(model.parameters())],
         iobytes=test_case.grad_iobytes(),
     )
+
+
+@pytest.mark.parametrize("executor", DEFAULT_EXECUTORS)
```

Comment:
DEFAULT_EXCUTORS does not include `thunder-torchcompile` yet. You may want to update that as well.

Reply:
good call.